demos/run_scripts.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -euo pipefail
+. "$(dirname "$0")"/helpers.sh
+
+# NOTE ON TOOLS: the only two nice tools I'm aware of that support our use-case
+# (non-animated svgs) are https://github.com/slowli/term-transcript and
+# https://github.com/FHPythonUtils/AnsiToImg. They have slightly different
+# limitations and advantages. If one of them stops being developed, we could
+# look at the other one.
+#
+# AnsiToImg can also generate PNGs, but that is currently harder to setup than
+# `convert`. `convert` supports different backends, but the simplest one with no
+# dependencies seemed to work for me. Another alternative for SVG -> PNG
+# conversion is Inkscape, which can be used directly or via `convert`.
+which term-transcript > /dev/null \
+  || echo '`term-transcipt` must be installed with e.g.'\
+          '`cargo binstall term-transcript-cli`.' \
+          'See also https://github.com/slowli/term-transcript'
+which convert > /dev/null \
+  || echo '`convert` from ImageMagick needs to be installed to create pngs.' \
+          'Only svgs will be created.' >&2
+
+echo "jj --version: (set PATH to change)"
+jj --version
+
+JJ_CONFIG=$(mktemp --tmpdir jjconfig-XXXX.toml)
+export JJ_CONFIG
+jj_config "$JJ_CONFIG"
+
+GIT_CONFIG_GLOBAL=$(mktemp --tmpdir gitconfig-XXXX)
+export GIT_CONFIG_GLOBAL
+git_config "$GIT_CONFIG_GLOBAL"
+
+for script in "$@"; do
+  script_base="${script%.sh}"
+  script_base="${script_base#demo_}"
+  # We use `term-transcript capture` instead of `term-transcript exec` so that
+  # we can show the output of the script via `tee`.
+  (bash "$script" || (echo "SCRIPT FAILED WITH EXIT CODE $?"; false)) 2>&1 | \
+    tee /dev/stderr | \
+    term-transcript capture --no-inputs --pure-svg --out "$script_base".svg "$script_base"
+  # Resize to 700 width and any height
```

Comment:
Some of the screenshots were 720px wide. Should we use that or 700px?

Reply:
That'd make them even taller, which bothers me a bit (though 20 pixels shouldn't matter much). 

I was thinking of later increasing the width both in terms of the number of pixels and the number of columns.

Reply:
I'm fine with making them even wider (I think my manually created screenshots were 815 pixels wide). I think I was just commenting on the fact that most (?) screenshots were 720 pixels wide but this line says 700.

Reply:
I think I see what you are saying: the SVG files have width 720 in whatever units SVG uses. I assumed that has nothing to do with pixels, but perhaps the fonts will be somehow clearer at 720; I'll try it out.

The PNG files have width 700 exactly AFAICT.

Reply:
Ah, I see. Yes, I just saw the width of 700 in the GitHub UI (maybe that's the SVG's suggested width in pixels?). So please ignore my comment.

Do feel free to increase the PNG width a little, however, if that helps reduce wrapping

Reply:
720 is, in fact, noticeably sharper. Also, it seems to be the default for 1 SVG unit to be 1 pixel, so I'll go with it.

Reply:
term-transcript just implemented https://github.com/slowli/term-transcript/pull/67, by the way, but it's not released yet. This would allow making the terminal more than 80 characters wide. I may or may not play with it, but I think 80 chars is fine for now.

Reply:
I realized that 80 characters look borderline OK on Mobile, but anything wider than that probably won't. So, I'm now thinking of sticking to 80 chars permanently.